opal/mca/common/ucx/common_ucx_wpool.h

```diff
@@ -441,7 +441,7 @@ opal_common_ucx_wpmem_post(opal_common_ucx_wpmem_t *mem, ucp_atomic_post_op_t op
                              len, rem_addr, rkey);
     if (OPAL_UNLIKELY(status != UCS_OK)) {
         MCA_COMMON_UCX_ERROR("ucp_atomic_post failed: %d", status);
-        rc = OPAL_ERROR;
+        return OPAL_ERROR;
```

Comment:
@janjust you can't return here without releasing the lock in the line 439

Reply:
good catch, fixed in forced push, for all 3 cases